### PR TITLE
Contractors  get a syndicate prisoner bag to transport someone to the extract point

### PR DESCRIPTION
**Pull request by Burger1998**
# Document the changes in your pull request
Contractor kit will now come with a syndicate prisoner transport bag
# Wiki Documentation
update the kit to include the bag in syndicate items
# Changelog
:cl:  
rscadd: Syndicate contractor will now get a prisoner bag to aid in kidnappings and keeping their targets alive
/:cl:

### CODE REVIEW

**Review by MajManatee — APPROVED** [review]
might get used now

File: code/game/objects/items/storage/uplink_kits.dm [hunk 1]
```diff
@@ -365,6 +365,7 @@
 	new /obj/item/modular_computer/tablet/syndicate_contract_uplink/preset/uplink(src)
 	new /obj/item/storage/box/syndicate/contractor_loadout(src)
 	new /obj/item/melee/classic_baton/telescopic/contractor_baton(src)
+	new /obj/item/bodybag/environmental/prisoner/syndicate
```
Comment (ReddicusDragon) [inline code/game/objects/items/storage/uplink_kits.dm]:
```suggestion
	new /obj/item/bodybag/environmental/prisoner/syndicate(src)
```

Comment (ReddicusDragon) [reply code/game/objects/items/storage/uplink_kits.dm]: without `(src)` at the end, it would not be spawned into the box.

**Review by ReddicusDragon — CHANGES_REQUESTED** [review]
Change this and it should be fine

**Review by ReddicusDragon — APPROVED** [review]

**Pull request merged**